src/channel_server/npc_packet.cpp

```diff
@@ -61,21 +62,18 @@ PACKET_IMPL(control_npc, const data::type::npc_spawn_info &npc, game_map_object
 	return builder;
 }
 
-PACKET_IMPL(animate_npc, packet_reader &reader) {
+PACKET_IMPL(animate_npc, game_map_object npc_id, uint8_t action1, uint8_t action2, const move_path* opt_path) {
```

Comment:
Pointer stars are affixed to the variable name rather than the type. For now.

If `move_path` is redesigned in the way I describe, it'll be passed by `const` reference like everything else.